packages/device_info_plus/device_info_plus/pubspec.yaml

```diff
@@ -33,7 +33,7 @@ dependencies:
   flutter_web_plugins:
     sdk: flutter
   meta: ^1.8.0
-  web: '>=0.5.0 <=0.6.0'
+  web: ">=0.5.0 <2.0.0"
```

Comment:
I am good with this, since it is not a breaking change.

Reply:
Awesome. Also wanted to suggest to have the range, so we don't get bombarded with issues :)

Reply:
I will do some testing in the meantime. If all good - will merge.